modules/commit-parser.js

```diff
@@ -8,7 +8,7 @@ const parseCommitMessage = (commitMsg, commitHash, settings) => {
   let result = {
     body: commitMsg,
     type: "",
```

Comment:
why is this change required?

Reply:
if the increase-version-if-release-none param is set to false, default release would be none and no version increase would be done. Therefore I changed default to patch, so if we cannot process commit message we ensure that version is increased. Ideally this would never happen with ensure conventional commits action put on PR, but you can always squash and change commit message so better to be on the safe side